cli/internal/cmd/upgradeapply.go

```diff
@@ -509,6 +536,21 @@ func parseUpgradeApplyFlags(cmd *cobra.Command) (upgradeApplyFlags, error) {
 	if skipHelmWait {
 		helmWaitMode = helm.WaitModeNone
 	}
+
+	rawSkipPhases, err := cmd.Flags().GetStringSlice("skip-phases")
+	if err != nil {
+		return upgradeApplyFlags{}, fmt.Errorf("parsing skip-phases flag: %w", err)
+	}
+	skipPhases := []skipPhase{}
```

Comment:
```suggestion
	var skipPhases []skipPhase
```
nit: [nil is a valid slice](https://github.com/edgelesssys/wiki/blob/master/company/gostyle.md#nil-is-a-valid-slice)